WinUIGallery/MainWindow.xaml

```diff
@@ -1,63 +1,55 @@
-<!--
-    //*********************************************************
-    //
-    // Copyright (c) Microsoft. All rights reserved.
-    // THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
-    // ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
-    // IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
-    // PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
-    //
-    //*********************************************************
--->
-<Page
-    x:Class="WinUIGallery.Pages.NavigationRootPage"
+<?xml version="1.0" encoding="utf-8" ?>
+<Window
+    x:Class="WinUIGallery.MainWindow"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    HighContrastAdjustment="None">
-
-    <Grid>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="Auto" />
-            <ColumnDefinition />
-        </Grid.ColumnDefinitions>
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIGallery"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:pages="using:WinUIGallery.Pages"
+    Title="MainWindow"
+    mc:Ignorable="d">
+    <Window.SystemBackdrop>
+        <MicaBackdrop />
+    </Window.SystemBackdrop>
+    <Grid x:Name="RootGrid" Loaded="RootGrid_Loaded">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
-            <!--  TitleBar  -->
             <RowDefinition Height="*" />
-            <!--  NavView  -->
         </Grid.RowDefinitions>
-
-        <Border
-            x:Name="AppTitleBar"
-            Grid.Column="1"
-            Height="{Binding ElementName=NavigationViewControl, Path=CompactPaneLength}"
-            Margin="48,0,0,0"
-            VerticalAlignment="Stretch"
-            AutomationProperties.AutomationId="AppTitleBar"
-            Canvas.ZIndex="1"
-            IsHitTestVisible="True">
-            <StackPanel VerticalAlignment="Stretch" Orientation="Horizontal">
-                <Image
-                    Width="18"
-                    VerticalAlignment="Center"
-                    Source="ms-appx:///Assets/Tiles/BadgeLogo.png" />
-                <TextBlock
-                    x:Name="AppTitle"
-                    Margin="12,0,0,0"
+        <TitleBar
+            x:Name="titleBar"
+            Title="WinUI 3 Gallery"
+            BackRequested="TitleBar_BackRequested"
+            IsBackButtonVisible="{x:Bind rootFrame.CanGoBack, Mode=OneWay}"
+            IsPaneToggleButtonVisible="True"
+            PaneToggleRequested="TitleBar_PaneToggleRequested"
+            Subtitle="Dev">
+            <TitleBar.IconSource>
+                <ImageIconSource ImageSource="/Assets/Tiles/GalleryIcon.ico" />
+            </TitleBar.IconSource>
+            <TitleBar.CenterContent>
```

Comment:
Hi @niels9001
The API names have been officially updated to LeftHeader, **Content**, and RightHeader.
https://github.com/microsoft/microsoft-ui-xaml/pull/10056#issuecomment-2672093972
Therefore, `<TitleBar.CenterContent>` should be changed to `<TitleBar.Content>` to reflect these updates.

Reply:
Yes, but those changes aren't part of Preview1 that I'm using in this PR..

Reply:
Updated to the latest version, and using the new property names.